qiskit/quantum_info/synthesis/one_qubit_decompose.py

```diff
@@ -327,6 +329,24 @@ def _circuit_u3(theta,
         circuit.append(U3Gate(theta, phi, lam), [0])
         return circuit
 
+    @staticmethod
+    def _circuit_u321(theta,
+                      phi,
+                      lam,
+                      phase,
+                      simplify=True,
+                      atol=DEFAULT_ATOL):
+        rtol = 1e-9
```

Comment:
Can you add a comment explaining the need for a custom `rtol` here?

Reply:
Will do. If we do a PR to use rtol on all the np.isclose calls, it will be covered there as well.

Reply:
Done in https://github.com/Qiskit/qiskit-terra/pull/5554/commits/b61b70954cbdd4b8719437057c9c6642441ac286